inst/shiny/server.R

```diff
@@ -230,6 +308,41 @@ shinyServer(function(input, output, session) {
   #-----------------------------------------------------------------------------
   # Page 3: DR & Clustering
   #-----------------------------------------------------------------------------
+  
+  #Plot dimensionality reduction plot
```

Comment:
In the new version of the code I don't think this event exists anymore. Can you double check and remove it from the pull request?